source/assets/css/components/_alerts.scss

```diff
@@ -5,7 +5,7 @@
   };
   text-align: center;
   background: darken($sl-color--hopbush, 10%);
-  color: rgba($sl-color--white, .875);
+  color: rgba($sl-color--white, 1);
```

Comment:
If we are gonna make this `1`, then we need to:

1. Not have this wrapped in the `rgba` function as it is pointless now. :)
2. Consider if the border on the links is enough now to differentiate them from the content, now that they are the same color.

Reply:
1. D'oh, yes, fixed—thanks for catching that. 😅
2. I removed the strong tag for better differentiation. How do you feel about this?
<img width="882" alt="Unbolded first sentence" src="https://user-images.githubusercontent.com/7484549/70948789-165dcb80-2011-11ea-8ddc-60c9c467cd1f.png">